server.js

```diff
@@ -1,19 +1,56 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import topMusicData from "./data/top-music.json";
+const listEndpoints = require('express-list-endpoints')
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+//Should I use docker for below issue or what do I need to do to solve it?
+//I couldn't make the MONGO_URL work with the localhost-version - when I did that it stopped working locally
+const mongoUrl = process.env.MONGO_URL || "mongodb+srv://ylva87:hHwxpsqpR3V62DS9@cluster0.hsexulf.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
The link you've added here goes in the environment variable, when you deploy on Google Cloud. It should not be in your codebase at all.